lib/net/http.rb

```diff
@@ -830,7 +830,7 @@ def HTTP.get_response(uri_or_host, path_or_headers = nil, port = nil, &block)
 
     # Posts data to a host; returns a Net::HTTPResponse object.
     #
-    # Argument +url+ must be a URL;
+    # Argument +url+ must be a URI;
```

Comment:
Minor: perhaps now this wording (used in many places) can be expanded to include "either as an instance of String or URI".